zeebe/util/src/main/java/io/camunda/zeebe/util/ReflectUtil.java

```diff
@@ -28,6 +29,22 @@ public static <T> T newInstance(final Class<T> clazz) {
     }
   }
 
+  /**
+   * Sets the field to be accessible via reflection if it is not currently for the given instance
+   * object. This replaces the old Junit 5 `ReflectUtils.makeAccessible` which was removed from
+   * their platform.
+   *
+   * @param field the field to make accessible
+   * @param instance the instance on which we check accessibility
+   * @return the field, accessible
+   * @param <T> the type of the field
+   * @param <U> the type of the instance, typically just {@code Object}
+   */
+  public static <T extends AccessibleObject, U> T makeAccessible(final T field, final U instance) {
```

Comment:
I removed the 
```java
if (!field.canAccess(object))
```
because it will throw if field is not declared inside object, so it will fail in most cases.

JUnit original code would check for modifiers, but it does really make a difference, considering that it's used only in test classes

Reply:
Let's do the same as them then :+1: Keep it identical.

As for `canAccess`, why would it fail in most cases? The expectation is you pass it an instance where the field is declared, no? 

Anyway, we could also omit the check, and just use `trySetAccessible` - it's a no-op when already true. If it returns false, we can throw the exception ourselves, or handle it in a way the test fails (throwing an `AssertionError` maybe?).

Reply:
Yes, for the field it would be ok, but the issue is for the method, which is defined in the `Closeable`, not in the test instance